Library/brew.sh

```diff
@@ -228,7 +228,8 @@ then
     -d av="$HOMEBREW_VERSION" \
     -d t=screenview \
     -d cd="$HOMEBREW_COMMAND" \
-    &
+    > /dev/null 2>&1 \
```

Comment:
Also `--output /dev/null` and `--silent` are already handling some of this but I'm :ok: with the duplication if you are.